ceph/datadog_checks/ceph/ceph.py

```diff
@@ -67,14 +70,26 @@ def _collect_raw(self, ceph_cmd, ceph_cluster, instance):
             name = cmd.replace(' ', '_')
             raw[name] = res
 
+        mon_map = raw['status']['monmap']
+        if 'min_mon_release_name' in mon_map and mon_map['min_mon_release_name'] == 'octopus':
+            self.log.debug("Detected octopus version of ceph...")
+            self._octopus = True
```

Comment:
Maybe reset at the beginning of each run?

Reply:
I dont think it can change from octopus to not octupus very easily but a reset when there is an error may be a good idea